models/quickbooks.yml

```diff
@@ -178,6 +178,54 @@ models:
       - name: recent_payment_date
         description: The last date a payment was provided for the bill or invoice.
 
+  - name: int_quickbooks__cash_flow_classifications
+    description: >
+      Table that provides additional classifications necessary for the cash flow statement, like the 
+      cash flow type and ordinals.
+
+  - name: quickbooks__cash_flow_statement
+    description: >
+      Table that provides all line items necessary to construct a cash flow statement.
+    columns:
+      - name: account_unique_id
+        description: Unique identifier of the cash flow statement account line, dependent on account_id, class_id, source_relation and calendar_date.
+        tests:
+          - unique
+          - not_null
```

Comment:
This test fails for me when I use the seed file. Does the same happen for you? It looks like we consistently get duplicates of 10 for the unique field. Why do you think that may be happening?

Reply:
Tests should be working now!